TELCODOCS-2153: Added row to the table for the rhel9 image (#90763)
openshift/openshift-docs: modules/gathering-data-specific-features.adoc

|Data collection for Local Storage Operator.

|`registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel8:v<installed_version_sandboxed_containers>`
|Data collection for {osc}.

|`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel8:v<installed_version_NHC>`
|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check (NHC) Operator, and the Node Maintenance Operator (NMO).

> Use this image if your NHC Operator version is *less than 0.9.0*. 

Reviewer: [Per ISG use (earlier) to refer to a previous software version or fix level](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#e).

Author: Thanks @mburke5678 for your feedback. I have applied your suggestion.

> For more information, see the "Gathering data" section for the specific Operator in https://docs.redhat.com/en/documentation/workload_availability_for_red_hat_openshift/latest/html/remediation_fencing_and_maintenance/index[Remediation, fencing, and maintenance] (Workload Availability for Red Hat OpenShift documentation).

Author: Thanks @mburke5678 for your feedback. Each Operator has its own _Gathering data_ section that is why I cannot point to a specific link. It depends on which Operator the reader wants to access.

|`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel9:v<installed_version_NHC>`
|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check (NHC) Operator, and the Node Maintenance Operator (NMO).

> Use this image if your NHC Operator version is *0.9.0. or greater*. 

Reviewer: [Per ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#l)

```suggestion
Use this image if your NHC Operator version is *0.9.0. or later*. 
```

> For more information, see the "Gathering data" section for the specific Operator in https://docs.redhat.com/en/documentation/workload_availability_for_red_hat_openshift/latest/html/remediation_fencing_and_maintenance/index[Remediation, fencing, and maintenance] (Workload Availability for Red Hat OpenShift documentation).

Reviewer: Do you have a direct link to the _Gathering data_ section?

|Data collection for {sandboxed-containers-first}.

|`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel8:v<installed-version-NHC>`
|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check (NHC) Operator, and the Node Maintenance Operator (NMO).

> Use this image if your NHC Operator version is less than 0.9.0. For more information, see the 'Gathering data' section for the specific Operator in the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.

Reviewer 2: Looks much better. NIT: Would it be better to highlight the difference between the two images using bold?
```suggestion
Use this image if your NHC Operator version is **less than 0.9.0**. For more information, see the 'Gathering data' section for the specific Operator in the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.
```

> Use this image if your NHC Operator version is *less than 0.9.0*. For more information, see the 'Gathering data' section for the specific Operator in the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.

Reviewer 3: I do not see gathering data section in the link https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift

> Use this image if your NHC Operator version is *less than 0.9.0*. For more information, see the 'Gathering data' section for the specific Operator in the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.

Author: Thanks @aravipra for your feedback. I have applied your suggestion.

> Use this image if your NHC Operator version is *less than 0.9.0*. For more information, see the 'Gathering data' section of the 'Remediation, fencing, and maintenance' User Guide for the specific Operator in the link:https://docs.redhat.com/en/documentation/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.

Reviewer 4: ```suggestion
Use this image if your NHC Operator version is *less than 0.9.0*. 
+
For more information, see the "Gathering data" section for the specific Operator in link:https://docs.redhat.com/en/documentation/workload_availability_for_red_hat_openshift[Remediation, fencing, and maintenance] (Workload Availability for Red Hat OpenShift documentation).
```

Reviewer 4: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute

Author: Thanks @agantony for your feedback. I have applied your suggestions.

|`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel9:v<installed-version-NHC>`
|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check (NHC) Operator, and the Node Maintenance Operator (NMO).

> Use this image if your NHC Operator version is 0.9.0. or greater. For more information, see the 'Gathering data' section for the specific Operator in the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.

Reviewer 2: Similar comment to the above one.
```suggestion
Use this image if your NHC Operator version is **0.9.0. or greater**. For more information, see the 'Gathering data' section for the specific Operator in the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.
```

Author: Thanks for your feedback @raz07. That's a good suggestion and I have applied it.

> Use this image if your NHC Operator version is *0.9.0. or greater*. For more information, see the 'Gathering data' section for the specific Operator in the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.

Reviewer 3: same comments as above

> Use this image if your NHC Operator version is *0.9.0. or greater*. For more information, see the section of the 'Remediation, fencing, and maintenance' User Guide for the specific Operator in the link:https://docs.redhat.com/en/documentation/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.

Reviewer 4: same as above

Author: Thanks @agantony for your feedback. I have made the changes as above.

|Data collection for {sandboxed-containers-first}.

|`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel8:v<installed-version-NHC>`
|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check (NHC) Operator, and the Node Maintenance Operator (NMO).

Use this image if your NHC Operator version is *less than 0.9.0*. 

> For more information, see the "Gathering data" section for the specific Operator in link:https://docs.redhat.com/en/documentation/workload_availability_for_red_hat_openshift[Remediation, fencing, and maintenance] (Workload Availability for Red Hat OpenShift documentation).

Reviewer 5: ```suggestion
For more information, see the "Gathering data" section for the specific Operator in https://docs.redhat.com/en/documentation/workload_availability_for_red_hat_openshift/latest/html/remediation_fencing_and_maintenance/index[Remediation, fencing, and maintenance] (Workload Availability for Red Hat OpenShift documentation).
```
the corrected link for that section of the docs

Author: Thanks @ShaunaDiaz for your feedback. It is really helpful. I have applied your suggestions.

|`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel9:v<installed-version-NHC>`
|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check (NHC) Operator, and the Node Maintenance Operator (NMO).

Use this image if your NHC Operator version is *0.9.0. or greater*. 

> For more information, see the "Gathering data" section for the specific Operator in link:https://docs.redhat.com/en/documentation/workload_availability_for_red_hat_openshift[Remediation, fencing, and maintenance] (Workload Availability for Red Hat OpenShift documentation).

Reviewer 5: ```suggestion
For more information, see the "Gathering data" section for the specific Operator in https://docs.redhat.com/en/documentation/workload_availability_for_red_hat_openshift/latest/html/remediation_fencing_and_maintenance/index[Remediation, fencing, and maintenance] (Workload Availability for Red Hat OpenShift documentation).
```
that's the correct link to that title in the doc

Author: Thanks @ShaunaDiaz for your feedback. It is really helpful. I have applied your suggestions.

|`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel8:v<installed-version-NHC>`
|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check Operator (NHC) Operator, and the Node Maintenance Operator (NMO) Operator.

Use this image if your NHC Operator version is less than 0.9.0. For more information, see the 'Gathering data' section for the specific Operator in the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.

|`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel9:v<installed-version-NHC>`

> |Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check Operator (NHC) Operator, and the Node Maintenance Operator (NMO) Operator.

Reviewer 2: Does it make sense to have `Operator` and then `Operator`? Here and on line 59
```suggestion
|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check Operator (NHC) Operator, and the Node Maintenance Operator (NMO).
```

Author: Thanks @razo7 for your feedback. I have applied your suggestion. I have also removed the first instance of Operator in NHC.
